packages/manager/apps/hycu/src/pages/dashboard/Dashboard.page.tsx

```diff
@@ -36,29 +42,61 @@ export type DashboardLayoutProps = {
   tabs: DashboardTabItemProps[];
 };
 
+const LicenseSuspendedBanner = () => {
+  const { t } = useTranslation('hycu/dashboard');
+
+  return (
+    <OsdsMessage className="mb-2" type={ODS_MESSAGE_TYPE.warning}>
+      <OsdsText
+        color={ODS_TEXT_COLOR_INTENT.warning}
+        size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+      >
+        {t('hycu_dashboard_warning_license_suspended_message')}
+      </OsdsText>
+    </OsdsMessage>
+  );
+};
+
+const LicenseErrorActivationBanner = ({
+  licenseHycu,
+}: {
+  licenseHycu: IHycuDetails;
+}) => {
+  const { t } = useTranslation('hycu/dashboard');
+
+  return (
+    <OsdsMessage className="mb-2" type={ODS_MESSAGE_TYPE.error}>
+      <OsdsText
+        color={ODS_TEXT_COLOR_INTENT.error}
+        size={ODS_THEME_TYPOGRAPHY_SIZE._400}
+      >
+        {t('hycu_dashboard_error_license_message', {
+          error: licenseHycu.comment,
+        })}
+      </OsdsText>
+    </OsdsMessage>
+  );
+};
+
 export default function DashboardPage() {
   const { serviceName } = useParams();
   const navigate = useNavigate();
   const { t } = useTranslation('hycu/dashboard');
-  const { addError, addWarning, clearNotifications } = useNotifications();
 
   const { data: licenseHycu } = useDetailsLicenseHYCU(serviceName);
   const { data: serviceDetails, error } = useServiceDetails({
     resourceName: serviceName,
   });
 
-  useEffect(() => {
+  const bannerPage = useMemo(() => {
     if (serviceDetails?.data.resource.state === 'suspended') {
-      clearNotifications();
-      addWarning(t('hycu_dashboard_warning_license_suspended_message'));
-    } else if (licenseHycu?.data.licenseStatus === LicenseStatus.ERROR) {
-      clearNotifications();
-      addError(
-        t('hycu_dashboard_error_license_message', {
-          error: licenseHycu.data.comment,
-        }),
-      );
+      return <LicenseSuspendedBanner />;
+    }
+    if (licenseHycu?.data.licenseStatus === LicenseStatus.ERROR) {
+      return <LicenseErrorActivationBanner licenseHycu={licenseHycu.data} />;
     }
+
+    return null;
   }, [licenseHycu, serviceDetails]);
```

Comment:
Why do you use a `useMemo` here instead of making a component ?

Reply:
I thought for the two method. But I don't think a method is better of the second. I don't known honnetly.